### PR TITLE
Add claims_snapshot.py tests

### LINKED ISSUE #389: Coverage: Increase coverage for claims_snapshot.py
https://app.codecov.io/gh/Badger-Finance/badger-rewards/blob/development/rewards/snapshot/claims_snapshot.py

### PR DESCRIPTION
**Pull request by SHAKOTN**
**Done:**
1. Raise proper ValueError in case `fetch_all_claimable_balances(chain)` returns None if badger API response was non-2xx
2. Some minor refactorings around
3. Lots of tests with real balance calculations against made up prices

Fixes #389

### CODE REVIEW

File: tests/test_snapshots/test_claims_snapshot.py [hunk 1]
```diff
@@ -0,0 +1,243 @@
...
+TEST_WALLET = '0xD27E9195aA35A7dE31513656AD5d4D29268f94eC'
+TEST_WALLET_ANOTHER = '0xF9e11762d522ea29Dd78178c9BAf83b7B093aacc'
+
+CLAIMABLE_BALANCES_DATA_ETH = {
+    'maxPage': 1,
+    'rewards': {
+        TEST_WALLET: [
+            {
+                'network': 'ethereum', 'address': BADGER,
+                'balance': '148480869281534217908'},
```
Comment (btcookies) [inline tests/test_snapshots/test_claims_snapshot.py:39]: this is formatted weird, is formatter installed?

Comment (SHAKOTN) [reply tests/test_snapshots/test_claims_snapshot.py:39]: I don't use autoformatter, but this is definitely PEP8 compatible. If it was a style problem I will see it by code analyzer. But I can fix that if this is a problem @btcookies ?

Comment (btcookies) [reply tests/test_snapshots/test_claims_snapshot.py:39]: If you npm install it'll run husky and black on commit

File: rewards/snapshot/claims_snapshot.py [hunk 2]
```diff
@@ -6,14 +6,16 @@
...
+from helpers.enums import BalanceType, Network
 from helpers.web3_utils import make_token
 from rewards.classes.Snapshot import Snapshot
 
 
 @lru_cache
 def claims_snapshot(chain: str) -> Dict[str, Snapshot]:
     all_claims = fetch_all_claimable_balances(chain)
+    if not all_claims:
+        raise ValueError("Badger-api returns error")
```
Comment (mitche50) [inline rewards/snapshot/claims_snapshot.py:18]: Do we want to be more descriptive with this error?  Maybe `Error retrieving claimable balances from Badger API`?

Comment (SHAKOTN) [reply rewards/snapshot/claims_snapshot.py:18]: Sure thing

File: tests/test_snapshots/test_claims_snapshot.py (hunk 1 above)
Comment (SHAKOTN) [reply tests/test_snapshots/test_claims_snapshot.py]: Fixed

File: rewards/snapshot/claims_snapshot.py (hunk 2 above)
Comment (SHAKOTN) [reply rewards/snapshot/claims_snapshot.py]: Fixed

**Review by btcookies — APPROVED** [review]
LGTM

**Review by mitche50 — APPROVED** [review]

**Pull request merged**